### PR TITLE
Setup rails backend

### PR DESCRIPTION
**Pull request by Kwentsir**
In this PR, the following were implemented;

> Created a new Rails API app called 'hello-rails-back-end'.

> Initialized project with Git.

> Postgres database set up.

> Created a table for storing messages and created 5 different greetings.

> Created an API endpoint that selects a random greeting from the table.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+To get a local copy up and running follow these simple example steps.
+
+1- Copy this link `https://github.com/Kwentsir/hello-rails-back-end.git` <br>
+2- Get the directory that you want to clone the repository. <br>
+3- Open the command prompt in this directory. <br>
+4- Write `git clone https://github.com/Kwentsir/hello-rails-back-end.git` <br>
+5- Go to the repository folder in your command prompt `cd hello-rails-back-end`. <br>
+6- You might need to install bundles with `bundle install` and npm dependencies with `npm i`
+6- Go to command line and run `./bin/dev` for the live server. <br>
```
Comment (alexander16108) [inline README.md:23]: In the README file for the frontend, you should provide a link to the Rails API (backend app).

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @Kwentsir , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
so close to making it on your first try 👍🏾 💯 🥇 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Kwentsir) [reply README.md:23]: Hello @alexander16108, thanks for your feedback. 
I think the link is already in there on line five of the readme.

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Kwentsir , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**